.github/workflows/merlin.yml

```diff
@@ -470,8 +470,10 @@ jobs:
       - name: Deploy merlin and mlp
         working-directory: merlin/scripts/e2e
         run: ./deploy-merlin.sh ${{ env.INGRESS_HOST }} ${{ env.LOCAL_REGISTRY }}:${{ env.LOCAL_REGISTRY_PORT }} ${{ needs.create-version.outputs.version }} ${{ github.ref }} ${{ env.MERLIN_CHART_VERSION }}
-      - name: Prune docker image to make some space
-        run: docker image prune --all --force
+      - name: Prune docker system to make some space
+        run: docker system prune --all --force
+      - name: Print space consumption
+        run: sudo df -h
```

Comment:
I don't see much point in keeping these here but I'm okay with leaving them too.

Reply:
leave it because its highly likely we need to shift the runner and these will be useful then